Documentation/kubernetes.md

```diff
@@ -47,14 +47,9 @@ rook-api-1709486253-gvdnc   1/1       Running   0          1m
 ```
 
 ### Provision Storage
-Before Rook can start provisioning storage, a StorageClass and its storage pool need to be created. This is needed for Kubernetes to interoperate with Rook for provisioning persistent volumes.  
+Before Rook can start provisioning storage, a StorageClass and its storage pool need to be created. This is needed for Kubernetes to interoperate with Rook for provisioning persistent volumes. The storage class sample will create the storage pool automatically. For more options on pools, see the documentation on [creating storage pools](pool-tpr.md).
```

Comment:
Ahh ok.. i see it now. Perhaps changing the documentation to "The rook-storageclass.yaml sample will create ...". That way there is no confusion that you are not referring to the storage class resource.